lms/lib/courseware_search/lms_result_processor.py

```diff
@@ -12,6 +13,8 @@
 
 from courseware.access import has_access
 
+UNNAMED_MODULE_NAME = _("(Unnamed)")
```

Comment:
I don't know if it's necessary to include a "Translators: " comment before every I18N string, but it seems to be something our translators look for to provide context around the term(s) they are being asked to translate.  Just thought I'd mention it.